### PR TITLE
:books: prepare changelog for 0.0.7

### PR DESCRIPTION
**Pull request by jayvdb**
Assumes #100 is merged.

### CODE REVIEW

File: changelog.yml [hunk 1]
```diff
@@ -1,6 +1,27 @@
 name: pypi-mobans
 organisation: moremoban
 releases:
+  - changes:
+      - action: Updated
+        details:
+          - sync templates/docs/make.bat.jj2 to include changes from
+            https://github.com/sphinx-doc/sphinx/commit/0dbdae31
+          - "#100: Fixed travis.yml.jj2 to not emit yamllint errors"
```
Comment (jayvdb) [inline changelog.yml]: I see I need to split this up into multiple action groups

Comment (jayvdb) [reply changelog.yml]: I split up https://github.com/moremoban/moban/pull/293 a bit more.  Is that better?

If so, I can split this one up.

Feel free to fiddle with them, if you think that will be faster.

Comment (chfw) [reply changelog.yml]: Actually this one looks great! :)

**Comment by chfw** [comment]
Would you like a release today?

**Pull request merged**